src/commons/components/catalog-search-panel/catalog-search-panel.tsx

```diff
@@ -1,14 +1,16 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 
 // CSS
 import './catalog-search-panel.css';
 
-export default class CatalogSearchPanel extends Component {
+
+
+export default class CatalogSearchPanel extends Component<{ onSearch: (stateValue:string) => void }, { value: string }>{
```

Comment:
export default class CatalogSearchPanel extends Component<Props, State>

Reply:
Description of properties and state to put in the interfaces on top of the class component
Exemple: 
interface Props {
  onSearch: (stateValue:string) => void
}

interface State {
  value: string
}